src/main/java/me/eccentric_nz/TARDIS/commands/tardis/TARDISVersionCommand.java

```diff
@@ -33,13 +38,27 @@
     }
 
     boolean displayVersion(CommandSender sender) {
-        String version = plugin.getPM().getPlugin("TARDIS").getDescription().getVersion();
-        String chunkversion = plugin.getPM().getPlugin("TARDISChunkGenerator").getDescription().getVersion();
+        final PluginManager pm = plugin.getPM();
+        List<String> hooks = plugin.getDescription().getSoftDepend();
+
+        String tardisversion = pm.getPlugin("TARDIS").getDescription().getVersion();
```

Comment:
`pm.getPlugin("TARDIS")` - don't need this - can be replaced with `plugin` - as `plugin` _is_ TARDIS